R/dettl.R

```diff
@@ -157,11 +178,37 @@ DataImport <- R6::R6Class(
       if (!force && !dry_run && !git_repo_is_clean(self$path)) {
         stop("Can't run load as repository has unstaged changes. Update git or run in dry-run mode.")
       }
-      run_load(private$con, private$load_, private$extracted_data,
-               private$transformed_data, private$test_queries,
-               private$load_pre_, private$load_post_, self$path,
-               private$load_test_, private$transaction, dry_run,
-               private$log_table, comment)
+
+      use_transaction <- private$load_in_transaction || dry_run
```

Comment:
This is the only place where `load_in_transaction` is used - perhaps drop it from the class and just use `dettl_config$dettl$transaction`

Reply:
We don't have the config at this point